Shared/ViewModels/SelectUserViewModel.swift

```diff
@@ -44,7 +44,6 @@ class SelectUserViewModel: ViewModel, Eventful, Stateful {
 
     var events: AnyPublisher<Event, Never> {
         eventSubject
-            .receive(on: RunLoop.main)
```

Comment:
Thank you for finding that material for the fix. I currently run all event subjects through the main thread generally as a practice but I guess I'll have to think about it more. This view is unfinished and the reason why this doesn't break on iOS is because in `SelectUserView` it will eventually send the action in a `Task { @MainActor in`, which then will send its corresponding event on the main actor for us to listen to.

I'm fine with this for now but think there's a different concurrency issue with `Stateful`.